core/src/types.ts

```diff
@@ -49,12 +51,17 @@ export type TTransformerOptions<Model, TransformedModel> = {
   addFields?: (args: { fields: Model }) => Partial<TransformedModel>;
   removeFields?: (keyof Model)[];
   replaceFields?: (args: { fields: Model }) => TransformedModel;
-  buildFields?: (keyof Model)[];
+  buildFields?: (keyof Model)[] | false;
```

Comment:
Nit: let's add a comment to explain what `false` refers to and when it's supposed to be used.

Reply:
Added here: af759dae62574ff496764d3fdcab22bd76f1126b